### DISCUSSION TITLE
How to use fields_default_sort to sort descending of a field?

### DISCUSSION THREAD

**Discussion by talischeung-ecg** [jowilf/starlette-admin#600] [Q&A] [answered]
I am trying to set the datatable default sorting ID descendingly.
Here is my code:

```
class UserActionModelView(ModelView):
    fields_default_sort = [("id", False)]
```

**Comment by jowilf** [comment, marked as answer]
As described [in the documentation ](https://jowilf.github.io/starlette-admin/api/views/?h=fields_default_sort#starlette_admin.views.BaseModelView), you need to set the second parameter to true for descending order
```py
   fields_default_sort = [("id", True)]
```
